qt/res/app.organicmaps.OrganicMaps.metainfo.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>app.organicmaps.OrganicMaps</id>
+  
+  <name>Organic Maps</name>
+  <summary>A free offline maps app for travelers, tourists, hikers, and cyclists based on top of crowd-sourced OpenStreetMap data and curated with love by MapsWithMe (Maps.Me) founders</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+  
+  <description>
+    <p>
+      # Organic Maps
+    </p>
+    <p>
+      [Organic Maps](https://organicmaps.app) is a free Android &amp; iOS offline maps app for travelers, tourists, hikers, and cyclists. It uses crowd-sourced OpenStreetMap data and is developed with love by <em>
```

Comment:
I’m not sure you really want to include Markdown in HTML

Reply:
Yeah, it looks like that generator did a pretty bad job. I went over the file and fixed that and other problems